cypress/component/annot-continuous-values.cy.js

```diff
@@ -7,18 +7,22 @@ const store = {
 
     getters: {
 
-        getHarmonizedPreview: () => (p_column, p_missingValue) => null,
-        getHeuristic: () => (p_column) => null,
+        getHarmonizedPreview: () => (p_column, p_missingValue) => {
+            return null;
```

Comment:
Curious, is this change for improving readability or is the `=> null` syntactically incorrect?

Reply:
I was having an issue with the test earlier, and was playing with the syntax to see if Cypress had a problem with this way of defining an arrow function. I believe the answer is that it does not. I can replace the original syntax.